apistar/core.py

```diff
@@ -69,8 +69,14 @@ def flatten_routes(routes: typing.Sequence[typing.Union[Route, Include]],
             path, method, view, name = item
             path = path_prefix + path
             name = namespace_prefix + name
-            route = Route(path, method, view, name)
-            flattened_routes.append(route)
+            if isinstance(method, (list, tuple)):
+                for _method in method:
+                    path_name = f"{_method}_{name}"
```

Comment:
F-strings only work on python 3.6. See: https://www.python.org/dev/peps/pep-0498/
But it seems like `apistar` aims to support python 3.5 too.

Reply:
nice catch. thanks.